dist/licensing/extras-files

```diff
@@ -1 +1 @@
-../../brooklyn-ui/src/main/license/source-inclusions.yaml:../../brooklyn-server/server-cli/src/main/license/source-inclusions.yaml
+../../brooklyn-ui/src/main/license/source-inclusions.yaml:../../brooklyn-server/server-cli/src/main/license/source-inclusions.yaml:../../brooklyn-client/src/main/license/source-inclusions.yaml
```

Comment:
(not part of this PR but it feels wrong to follow the `/src/main/` structure for a go project and easy to change to put this info into `build/`)